javascript/javascript1/week2/weather_wear.js

```diff
@@ -0,0 +1,16 @@
+function clothesToWear(degree) {
+    if (degree < -10) {
+      return "Winter jacket and warm pants + thermal underwear"
+    }else if (degree < 0 && degree >= -10) {
```

Comment:
Tip: You can omit the second check (`degree >= -10`) because it will always be true at this point.

```js
function clothesToWear(degree) {
    if (degree < -10) {
      return "Winter jacket and warm pants + thermal underwear"
    }else if (degree < 0) { // If our program hits this line we already know degree >= -10
    ...
```